scripts/zkai_ezkl_statement_envelope_benchmark.py

```
@@ -0,0 +1,511 @@
+    writer.writeheader()
+    for case in payload["cases"]:
+        writer.writerow({column: case[column] for column in TSV_COLUMNS})
+    return output.getvalue()
+
+
+def main(argv: list[str] | None = None) -> int:
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument(
+        "--srs-path",
+        type=pathlib.Path,
+        default=pathlib.Path(os.environ.get("ZKAI_EZKL_SRS_PATH", "/tmp/ptvm-ezkl-kzg17.srs")),
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/ptvm-ezkl-kzg17.srs"

(S108)